oneview/resource_logical_enclosure.go

```diff
@@ -82,15 +83,18 @@ func resourceLogicalEnclosure() *schema.Resource {
 
 			"enclosure_group_uri": {
 				Type:     schema.TypeString,
-				Required: true,
+				Computed: true,
+				Optional: true,
 			},
 			"enclosure_uris": {
```

Comment:
This is a read only field. Therefore, we can remove optional.
And if oneview is not sending an ordered array, then making this attribute to TypeSet would be a good idea.

Reply:
done